From 0a5c9e6e1ac4b8fb90d2c557ae5ef80d5be5ebae Mon Sep 17 00:00:00 2001
From: Jasmin Jessich <jasmin@anw.at>
Date: Sun, 6 May 2018 12:03:30 +0200
Subject: [PATCH] media: Use ktime_set() in pt1.c

In commit 20a63349b142 a new variable ktime_t delay has been added.
We decided to use the API functions to initialize ktime_t variables
within media-tree. Thus variable delay needs to be initialized with
ktime_set() instead of setting it directly.

Signed-off-by: Jasmin Jessich <jasmin@anw.at>
---
 drivers/media/pci/pt1/pt1.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/media/pci/pt1/pt1.c b/drivers/media/pci/pt1/pt1.c
index 55a89ea..5708f69 100644
--- a/drivers/media/pci/pt1/pt1.c
+++ b/drivers/media/pci/pt1/pt1.c
@@ -485,7 +485,7 @@ static int pt1_thread(void *data)
 		if (!pt1_filter(pt1, page)) {
 			ktime_t delay;
 
-			delay = PT1_FETCH_DELAY * NSEC_PER_MSEC;
+			delay = ktime_set(0, PT1_FETCH_DELAY * NSEC_PER_MSEC);
 			set_current_state(TASK_INTERRUPTIBLE);
 			schedule_hrtimeout_range(&delay,
 					PT1_FETCH_DELAY_DELTA * NSEC_PER_MSEC,
-- 
2.7.4

